kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml

```diff
@@ -428,6 +428,12 @@ webApi:
     table: biorxiv_medrxiv_meca_path_metadata
     dataUrl:
       urlExcludingConfigurableParameters: https://api.biorxiv.org/meca_index/elife/all
+    airflow:
+      dagParameters:
+        schedule: '@hourly'
```

Comment:
This seems to be the only one changed to hourly.
Did we have cases where it daily wasn't enough?
(Also asking because I believe this is `append` without a date filter and will therefore create more duplicate records in BigQuery)